src/components/contestants.jsx

```diff
@@ -0,0 +1,216 @@
+            <Header isLoggedIn={this.props.isLoggedIn} />
+            <Row className="xsmall-spacing" />
+            <Row>
+              <Col>
+                <h1>Participanți InfoEducație Ediția 2015</h1>
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <h2>Concurs Național de Informatică</h2>
+              </Col>
+            </Row>
+            <Row className="big-spacing" />
+          </Grid>
+        </div>
+        <Grid className="stats-section">
+          <Row>
+            <Col md={6} mdOffset={3}
+                 sm={8} smOffset={2}
+                 xs={12}>
+              <Row className="inner-stats">
+                <Col xs={4}>
+                    <p className="description">Participanți</p>
+                    <p className="value">75</p>
+                </Col>
+                <Col xs={4} className="border-left">
+                    <p className="description">Proiecte</p>
+                    <p className="value">39</p>
+                </Col>
+                <Col xs={4} className="border-left">
+                    <p className="description">Județe</p>
+                    <p className="value">17</p>
+                </Col>
+              </Row>
+            </Col>
+          </Row>
+        </Grid>
+        <div className="white-section-wrapper">
+          <Grid>
+            <Row className="big-spacing" />
+            <Row className="white-section">
+              <Col mdOffset={2} md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="educational" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="media" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="robots" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="utility" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="web" />
+              </Col>
+              <Col mdOffset={2} md={1}>
+                <Glyphicon glyph="th-large"
+                           className={gridClassName}
+                           onClick={this.showGrid} />
+              </Col>
+              <Col md={1}>
+                <Glyphicon glyph="align-justify"
```

Comment:
Scoate `hidden-hs`. Facem din React aia default si in teorie el nu va mai vedea filtrele.